assets/js/translations-page.js

```diff
@@ -45,8 +45,10 @@ jQuery(function($) {
 	});
 
 	var lastClicked = false;
-	// Check all checkboxes from WP common.js, synced with [20400]
+	// Check all checkboxes from WP common.js, synced with [25141]
 	$('tbody').children().children('.checkbox').find(':checkbox').click( function(e) {
+		var checks, first, last, sliced;
+
```

Comment:
line can be removed

Reply:
Can be, but it's more readable to separate the vars from the code.  When the file's min'd it will get deleted anyway.

Reply:
Good point